### DISCUSSION TITLE
Import Bug

### DISCUSSION THREAD

**Discussion by sbonaime** [requarks/wiki#3687] [Error / Bug Report]
After a fresh install of 2.0.0-rc.17, I set the email settings and change the locale to Fr. 

At the end of the "Import from wiki.js 1.x" without importing users, I have the following error :

<img width="752" alt="Capture d’écran 2019-10-29 à 29-10-19 15 20 46" src="https://user-images.githubusercontent.com/750709/67776044-0d646b00-fa60-11e9-94ec-efb4587bb0fe.png">
I can see that data has been uploaded from git in the repo directory but pages are not visible in the "Pages" section

If I "Start Import" a second time without reloading the page, the import is successful

**Comment by Cylix** [comment]
Got similar issue while importing on my own. It spins for a while (probably 2 or 3 minutes), then I got a `Network erorr: failed to fetch` error.

It happens every single time I attempt to import.

However, looking at the server logs, I can see the import task is still on-going and will eventually succeed.
